pkg/k8shandler/secret_test.go

```diff
@@ -0,0 +1,67 @@
+package k8shandler
+
+import (
+	"reflect"
+	"testing"
+
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+    rbac "k8s.io/api/rbac/v1"
```

Comment:
the formatting is off here and elsewhere

Reply:
:p  Thanks!